utils/target_system.mli

```diff
@@ -41,3 +41,38 @@ type assembler =
   | MASM
 
 val assembler : unit -> assembler
+
+
+type machine_width =
+  | Thirty_two
+  | Sixty_four
+
+
+(** The natural machine width of the target system. *)
+val machine_width : unit -> machine_width
+
+
```

Comment:
Remove double blank lines.  I would also leave a CR saying that we should cut down this list of targets, as some of them are obscure.  OK for now though.